plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -2,6 +2,67 @@
 <?eclipse version="3.4"?>
 <plugin>
 
+  <extension point="org.eclipse.ui.commands">
+    <command
+      description="%deployStandardCommandDescription"
+      id="com.google.cloud.tools.eclipse.appengine.deploy"
+      name="%deployStandardCommandName">
+    </command>
+  </extension>
+
+  <extension point="org.eclipse.ui.menus">
+    <menuContribution
+      allPopups="false"
+      locationURI="popup:org.eclipse.ui.popup.any?after=additions">
+      <command
+        commandId="com.google.cloud.tools.eclipse.appengine.deploy"
+        label="%deployStandardMenuLabel"
+        style="push">
+        <visibleWhen checkEnabled="true" />
+      </command>
+    </menuContribution>
+  </extension>
+
+  <extension point="org.eclipse.ui.handlers">
+    <handler
+      class="com.google.cloud.tools.eclipse.appengine.deploy.ui.standard.StandardDeployCommandHandler"
+      commandId="com.google.cloud.tools.eclipse.appengine.deploy">
+      <enabledWhen>
+        <and>
+          <count value="1" />
+          <iterate>
+            <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
+          </iterate>
+        </and>
+      </enabledWhen>
+    </handler>
+  </extension>
+
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
```

Comment:
Domain names have a very clamped down character space.  There were some performance issues in the Console view with very long lines that were traced to an [overly broad regex](https://bugs.eclipse.org/bugs/show_bug.cgi?id=175888#c29) to search for Java exception stacktraces.